pkg/config/global.go

```diff
@@ -120,50 +122,80 @@ func LoadGlobalConfig(configFile string) error {
 
 // GetPolicyBasePath returns the configured policy base path
 func GetPolicyBasePath() string {
+	if global == nil {
```

Comment:
Can there be a case where `global` is not nil? But, `global.Policy.BasePath`is an empty string?